DEPLOYMENT.md

```diff
@@ -0,0 +1,100 @@
+This file gives instructions for deploying the API to AWS, and the app to TestFlight (iOS) and the Google Play Store (Android).
+
+- [Setup to deploy the app](#setup-to-deploy-the-app)
+- [API deployment on AWS](#api-deployment-on-aws)
+- [App deployment](#app-deployment)
+
+# Setup to deploy the app
+
+This is how you get set up ready to deploy the app to the Google Play Store (for Android) and TestFlight (for iOS) later, using [Fastlane](https://fastlane.tools/).
+
+You don't need to worry about doing this section until you're through all the steps above and you've solved any headaches getting the API and the app running locally.
```

Comment:
Tiny thing, but "all the steps above" should probably be something like "all the setup steps in the README"?